internal/lang/funcs/sensitive.go

```diff
@@ -60,10 +60,34 @@ var NonsensitiveFunc = function.New(&function.Spec{
 	},
 })
 
+// IsSensitiveFunc returns whether or not the value is sensitive.
+var IsSensitiveFunc = function.New(&function.Spec{
+	Params: []function.Parameter{
+		{
+			Name:             "value",
+			Type:             cty.DynamicPseudoType,
+			AllowUnknown:     true,
+			AllowNull:        true,
+			AllowMarked:      true,
+			AllowDynamicType: true,
+		},
+	},
+	Type: func(args []cty.Value) (cty.Type, error) {
+		return cty.Bool, nil
+	},
+	Impl: func(args []cty.Value, retType cty.Type) (ret cty.Value, err error) {
+		return cty.BoolVal(args[0].HasMark(marks.Sensitive)), nil
```

Comment:
HasMark can only check if the Value itself has mark
I am not sure if the Value is a list that contains some sensitive value, what should it return?
@cube2222 @janosdebugs which does the core team prefer?

Reply:
I think checking for the mark is fine, and what is expected here. It's how it's handled everywhere else, too.